src/http/representation/RepresentationMetadata.ts

```diff
@@ -330,4 +379,16 @@ export class RepresentationMetadata {
       this.set(CONTENT_LENGTH_TERM, toLiteral(input, XSD.terms.integer));
     }
   }
+
+  /**
+   * Shorthand for the ContentType as an object (with parameters)
+   */
+  public get contentTypeObject(): ContentType | undefined {
+    return this.contentType ?
+      {
+        value: parseContentType(this.contentType).type,
+        parameters: this.getContentTypeParams(),
+      } :
```

Comment:
And then this could be changed to just be the result of the `getContentType` function which returns the entire object. Its output type could then be simplified to `ContentType`.